virt/virt-4-8-release-notes.adoc

```diff
@@ -214,3 +214,38 @@ No model registered for VirtualMachines
 ----
 +
 You can ignore this message. It does not affect VM import. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1974812[*BZ#1974812*])
+
+* When creating a virtual machine after upgrading to OCP 4.8 while running CNV 2.6.5, various errors occur.
++
+** Selecting *Create* -> *With YAML* on the *Virtualization* page of the {VirtProductName} console displays the following warning message:
++
+[source]
+----
+The server doesn't have a resource type "kind: VirtualMachine, apiVersion: kubevirt.io/v1"
+----
++
+*** As a workaround, modify the `apiVersion` in the manifest. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1979114[*BZ#1979114*])
++
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1alpha3 <1>
+kind: VirtualMachine
+metadata:
+  annotations:
+...
+----
+<1> The modified `apiVersion: kubevirt.io/v1alpha3`
++
+** Creating a virtual machine from the Customize wizard provides a unclear and non-specific error message:
++
+[source]
+----
+Error creating virtual machine
+----
++
+*** As a workaround, copy the manifest and create the virtual machine from the CLI. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1979116[*BZ#1979116*])
++
+// fix targeted for 4.8.1
+** When connecting to the VNC console by using the {VirtProductName} web console and `virtctl` command, the VNC console always fails to respond. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1977037[*BZ#1977037*])
++
+*** As a workaround, create the virtual machine from the CLI or upgrade to CNV 4.8. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1979116[*BZ#1979116*])
```

Comment:
The bug number should be BZ#1977037, isn't it?

Reply:
Yes, you are correct. I made a copy/paste error that included the 1977037 number. It's been removed.